server/app/views/ViewUtils.java

```diff
@@ -75,10 +75,14 @@ LinkTag makeLocalCssTag(String filename) {
         .withRel("stylesheet");
   }
 
-  public ImgTag makeLocalImageTag(String filename) {
+  public ImgTag makeLocalImageTagPNG(String filename) {
     return img().withSrc(assetsFinder.path("Images/" + filename + ".png"));
```

Comment:
I would actually keep `makeLocalImageTag` name and instead passed filename with extension:

.makeLocalIImageTag("ChiefSeattle_Blue.png")

I don't think that passing name without extension saves much space.

Reply:
This is a non-issue now that we're not using webpack